docs/UserGuide.md

```diff
@@ -48,28 +64,48 @@ Type the following command: **`java -jar agentassist.jar`** and press **Enter**.
 * A window similar to the below image should appear in a few seconds. You will see a graphical user interface with sample contact information already added.<br>
   ![Ui](images/Ui.png)
 
-### 6. Using AgentAssist
+---
+
+## Using AgentAssist {#using-agentassist}
+
+AgentAssist's command box is your portal to managing contacts efficiently. Here’s how you can use it to streamline your tasks:
+
+* To get started, simply type a command into the command box and hit **Enter**.
+
+Some initial commands to try:
+
+
+- **Viewing Contacts**
+  * `list`: This command displays all contacts currently in your database, making it easy to browse through entries.
```

Comment:
Not sure if this is needed, as it is currently implied, but maybe we could specify the list indexes (starting from 1) from the list view are used for all other commands? Perhaps this could be more specific for the user guide